src/charm.py

```diff
@@ -117,18 +117,15 @@ def on_start(self, event):
         if not self.unit.is_leader():
             return
 
-        # FIXME: This is awful!!! Only for development purpouses.
-        # We need to find a better way to wait for MySQL pod is ready
-        import time
-        time.sleep(40)
-
         if not self._mysql_is_ready():
             message = "Waiting for MySQL Service"
             self.unit.status = WaitingStatus(message)
             logger.info(message)
             event.defer()
         else:
-            self.unit.status = ActiveStatus("MySQL Service is ready")
+            message = "MySQL Service is ready"
+            self.unit.status = ActiveStatus(message)
```

Comment:
Minor Point. The generally recommendation is not to set a message for `ActiveStatus`. Some exceptional cases may be warning messages.